packages/xrpl/src/sugar/submit.ts

```diff
@@ -248,8 +182,42 @@ function isSigned(transaction: Transaction | string): boolean {
   return tx.SigningPubKey != null && tx.TxnSignature != null
 }
 
-// initializes a transaction for a submit request
-async function getSignedTx(
+/**
+ * Retrieves the signed transaction from the client.
```

Comment:
We don't want to give the impression that we're doing something which could reveal secrets online. The signing happens offline, but the client is used to update the transaction before that.
```suggestion
 * Updates a transaction with `autofill` then signs it if it is unsigned.
```